pyproject.toml

```diff
@@ -28,16 +28,20 @@ types-pyyaml = "^6.0.12.12"
 jsonschema = "^4.2.0"
 types-jsonschema = "^4.2.0"
 prompt-toolkit = "^3.0.39"
-pinecone-text = "^0.7.2"
```

Comment:
Found this syntax [here](https://github.com/python-poetry/poetry/issues/5733#issuecomment-1151481078). 
Was skeptical at first - but it seems to be working